pkg/cri/config/config_unix.go

```diff
@@ -110,5 +110,6 @@ func DefaultConfig() PluginConfig {
 		EnableCDI:                false,
 		CDISpecDirs:              []string{"/etc/cdi", "/var/run/cdi"},
 		ImagePullProgressTimeout: time.Minute.String(),
+		DrainExecSyncIOTimeout:   "0",
```

Comment:
a user won't know if that is 0ms or 0s then fail if they set it to a number > 0 ... maybe 0s to make it make more sense/more obvious?

Reply:
Sure. `0s` will be good. Updated